src/group.h

```diff
@@ -24,14 +24,13 @@ extern GroupPool _group_pool; ///< Pool of groups.
 struct GroupStatistics {
 	Money profit_last_year;                 ///< Sum of profits for all vehicles.
 	Money profit_last_year_min_age;         ///< Sum of profits for vehicles considered for profit statistics.
-	uint16_t *num_engines;                    ///< Caches the number of engines of each type the company owns.
+	std::vector<uint16_t> num_engines; ///< Caches the number of engines of each type the company owns.
```

Comment:
All these comment not aligned :)

Reply:
Good! Let's stop trying to align them! :D